### DISCUSSION TITLE
problem of prepare_data.sh

### DISCUSSION THREAD

**Discussion by Y3224235216** [wenet-e2e/wespeaker#324] [Q&A]
I encountered a problem with prepare_data.sh that shows "Cannot open directory: No such file or directory at ./local/m4a2wav.pl line 27. cat: /home/ma-user/work/wespeaker/examples/voxceleb/v2/data/raw_data/voxceleb2_wav/dev/m4a2wav_dev.sh: No such file or directory". Obviously, this file exists. How can I solve this problem? (This is my first time commenting, so I hope I'm not doing it wrong.) Please help me!

**Comment by JiJiJiang** [comment]
You can refer to our [docs](https://wenet.org.cn/wespeaker/vox.html).
